lib/sml-conv/conversionals.sig

```diff
@@ -1,14 +1,47 @@
+(* This the payoff for creating a CONV ascribing structure. This signature
+ * specifies several "generic" conversions as well as ways to combine
+ * existing ones.
+ *)
 signature CONVERSIONALS =
 sig
   type conv
 
+  (* CID is the conversion which always succeeds but does nothing.
+   * It simply returns the supplied term with no modifications
+   *)
   val CID : conv
+
+  (* CTHEN (t1, t2) will take a term, apply t1 to it and then apply t2
```

Comment:
CTHEN (t1, t2) will run t2 on the result of t1. If either fails, the entire conv will fail.